OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -321,7 +322,7 @@
     <Compile Include="Traits\Player\PlaceBeacon.cs" />
     <Compile Include="Traits\Player\PlaceBuilding.cs" />
     <Compile Include="Traits\Player\PlayerStatistics.cs" />
-    <Compile Include="Traits\Player\ProductionQueue.cs" />
+    <Compile Include="Traits\Buildings\ProductionQueue.cs" />
```

Comment:
What do you mean? I see only TD use `ProductionQueue` and that is on the factories, not the players.
Yeah that is a good idea.